packages/netlify-cms-backend-github/src/implementation.js

```diff
@@ -86,7 +88,7 @@ export default class GitHub {
     var repoExists = false;
     while (!repoExists) {
       repoExists = await fetch(`${this.api_root}/repos/${repo}`, {
-        headers: { Authorization: `Bearer ${token}` },
```

Comment:
BUG: Looks like a bug introduced in https://github.com/netlify/netlify-cms/pull/2430

The correct way to call the API is using `token ${token}` (this is done properly in `API.js`)

Reply:
Fixed in #2523: https://github.com/netlify/netlify-cms/pull/2523/commits/1bedf90a267a45e07d1e343f550df73846e46eb7